### DISCUSSION TITLE
blog/2024-12-19-semantic-version-for-nextjs-app

### DISCUSSION THREAD

**Discussion by giscus** [mazipan/blog-comments#24] [Q&A]
# blog/2024-12-19-semantic-version-for-nextjs-app

Bagaimana menambahkan semantic version untuk aplikasi Next.js

https://www.mazipan.space/blog/2024-12-19-semantic-version-for-nextjs-app

**Comment by mazipan** [comment]
💬 Leave your comment, if you have another tips and tricks, or when you have a different point of view about the topic.

Use polite sentences, do not harm anyone by using aggressive words. 🙏

It is okay to have different perspective.
